build/IPDK_Container/scripts/build_p4sde.sh

```diff
@@ -19,11 +20,11 @@ git config --global url."https://github.com/".insteadOf git@github.com:
 git config --global url."https://".insteadOf git://
 
 WORKDIR=$1
-cd ${WORKDIR}
+cd "${WORKDIR}" || exit
 
 echo "Removing p4-sde directory if it already exists"
 if [ -d "p4-sde" ]; then rm -Rf p4-sde; fi
-mkdir $1/p4-sde && cd $1/p4-sde
+mkdir "$1"/p4-sde && cd "$1"/p4-sde || exit
```

Comment:
File level error flag "set -e" enabled i.e. exit if any command returns non-zero exit status.
Is there any specific reason for adding command level exit.

Reply:
Spellcheck requires this.